nav2_behavior_tree/plugins/condition/goal_updated_condition.cpp

```diff
@@ -0,0 +1,76 @@
+#ifndef NAV2_BEHAVIOR_TREE__GOAL_UPDATED_CONDITION_HPP_
+#define NAV2_BEHAVIOR_TREE__GOAL_UPDATED_CONDITION_HPP_
```

Comment:
hmm I think uncrustify wasn't complaining on this one, I run it through and it only complaint about the copyright, but I'll check it again. If you say it because of the `HPP`, I also thought maybe it should be called `NAV2_BEHAVIOR_TREE__GOAL_UPDATED_CONDITION_CPP_`, but then all other plugins are using HPP instead of CPP on the guard (even when they are cpp files). Is that what you saw?

Reply:
Just see what it says when you fix the copyright and work from there. Maybe I'm mistaken.

Reply:
For sure, thanks.